pkg/remote/google/repository/asset_test.go

```diff
@@ -52,10 +52,10 @@ func Test_assetRepository_searchAllResources_CacheMiss(t *testing.T) {
 		t.Fatal(err)
 	}
 	c := &cache.MockCache{}
-	c.On("GetAndLock", "SearchAllResources").Return(nil).Times(1)
-	c.On("Unlock", "SearchAllResources").Times(1)
-	c.On("Put", "SearchAllResources", mock.IsType([]*assetpb.ResourceSearchResult{})).Return(false).Times(1)
-	repo := NewAssetRepository(assetClient, config.GCPTerraformConfig{Scope: []string{""}}, c)
+	c.On("GetAndLock", "SearchAllResources_").Return(nil).Times(1)
+	c.On("Unlock", "SearchAllResources_").Times(1)
+	c.On("Put", "SearchAllResources_", mock.IsType([]*assetpb.ResourceSearchResult{})).Return(false).Times(1)
+	repo := NewAssetRepository(assetClient, config.GCPTerraformConfig{Scopes: []string{""}}, c)
```

Comment:
Is there a particular reason to use an empty scope here ? I find it confusing to read

Reply:
The only reason I kept it empty was because previously it also had empty project, region, etc. Now I've added a dummy scope